js/client-teambuilder.js

```diff
@@ -1202,12 +1202,12 @@
 				}
 				for (i = 0; i < this.curSetList.length; i++) {
 					if (this.curSetList.length < this.curTeam.capacity && this.deletedSet && i === this.deletedSetLoc) {
-						buf += '<li><button name="undeleteSet"><i class="fa fa-undo"></i> Undo Delete</button></li>';
+						buf += '<li><button name="undeleteSet" class="button"><i class="fa fa-undo"></i> Undo Delete</button></li>';
```

Comment:
This looks better as not a button...

Reply:
Would you prefer to keep it white? 
![image](https://github.com/smogon/pokemon-showdown-client/assets/17451256/35ff7364-dca6-4d94-9423-b1124deb9f03)
![image](https://github.com/smogon/pokemon-showdown-client/assets/17451256/ae86e9d1-6bf5-46b5-953a-3565c7389e10)

Reply:
It shouldn't be white. I must be mixing this up with something else.

Reply:
Yeah, I got this mixed up with undeleting teams, which already has styling.